src/palette/internal/paletteuiactions.cpp

```diff
@@ -44,6 +44,12 @@ const UiActionList PaletteUiActions::m_actions = {
              TranslatableString("action", "Palette search"),
              TranslatableString("action", "Search palettes")
              ),
+    UiAction("apply-current-palette-element",
+             mu::context::UiCtxNotationOpened,
+             mu::context::CTX_ANY,
```

Comment:
Perhaps this should be CTX_NOTATION_OPENED?  Doesn't make sense to run this command without a score, but I don't understand the difference between the two contexts here

Reply:
Indeed, CTX_NOTATION_OPENED makes sense here. The first context determines when an action is enabled (although there may be additional logic for enabled-ness), the second one is the shortcut context, which determines when the keyboard shortcut can be activated. This is useful when the same shortcut is used for different actions in different contexts.